### ISSUE TITLE
Vouchers being zero valued if combined and potentially over order total

### ISSUE THREAD

**Issue by gazlawn22** [verbb/gift-voucher#81]
A customer has added items to their cart to a value of £298.81.

They then apply 4 voucher codes to their order, 1 for £150, and 3 for for £50.00 (total of £300). This should then leave a voucher credit of £1.19 (not sure which voucher code it would leave the credit on).

Instead, as soon as the voucher code is applied that will take it into a payment requiring 'zero', it's sets the value of that voucher to zero, or depending on the order the voucher was applied, its changes the £150 value voucher to £99.

Essentially forcing the user to have to pay something even though the applied vouchers should cover the cost?

![vouchers-example](https://user-images.githubusercontent.com/70768911/145191207-ed707c12-d493-4121-bc85-005231f2cc29.jpg)

**Additional info**
- Plugin version: 2.6.3
- Craft version: 3.7.25.1
- Multi-site: No

**Additional context**

Config: 'registerAdjuster' => 'afterTax',
Commerce: Minimum Total Price Strategy: Default - Allow the price to be negative if discounts are greater than the order value.

**Comment by engram-design** [comment]
I'm testing this on the `dev-craft-3` branch (release imminent) and things seem to be applying correctly for me, with your exact scenario. The last voucher should indeed have $1.19 left on it.

If you're able to, would love to see if you can test the latest code - change your `verbb/gift-voucher` requirement in `composer.json` to:

```
"require": {
  "verbb/gift-voucher": "dev-craft-3 as 2.6.3",
  "...": "..."
}
```

Then run `composer update`.

**Comment by gazlawn22** [comment]
Thanks Josh, yeah, that looks as if that works now! Just testing on local.

Thanks!

**Comment by engram-design** [comment]
Fixed in [2.7.0](https://github.com/verbb/gift-voucher/releases/tag/2.7.0)
